iommi/locale/sv/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,172 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
```

Comment:
More boilerplate removal...

Reply:
This is added by the tool, and is readded if I remove it.